corehq/apps/userreports/tasks.py

```diff
@@ -97,3 +103,47 @@ def _iteratively_build_table(config, last_id=None, resume_helper=None):
             if config.meta.build.initiated == current_config.meta.build.initiated:
                 current_config.meta.build.finished = True
                 current_config.save()
+
+
+@task(queue='ucr-queue')
+def compare_ucr_dbs(domain, report_config_id, filter_values, sort_column, sort_order, params):
+    from corehq.apps.userreports.laboratory.experiment import UCRExperiment
+
+    def _run_report(backend_to_use):
+        data_source = ReportFactory.from_spec(spec, include_prefilters=True)
+        data_source.override_backend_id(backend_to_use)
+        data_source.set_filter_values(filter_values)
+        if sort_column:
+            data_source.set_order_by(
+                [(data_source.top_level_columns[int(sort_column)].column_id, sort_order.upper())]
+            )
+
+        datatables_params = DatatablesParams.from_request_dict(params)
+        page = list(data_source.get_data(start=datatables_params.start, limit=datatables_params.count))
+        total_records = data_source.get_total_records()
+        json_response = {
+            'aaData': page,
+            "iTotalRecords": total_records,
+            "iTotalDisplayRecords": total_records,
+        }
+        # these are to be checked after ES supports total rows
+        # total_row = data_source.get_total_row() if data_source.has_total_row else None
+        # if total_row is not None:
+        #     json_response["total_row"] = total_row
+        return json_response
+
+    spec = get_document_or_not_found(ReportConfiguration, domain, report_config_id)
+    experiment_context = {
+        "domain": domain,
+        "report_config_id": report_config_id,
+        "filter_values": filter_values,
+    }
+    experiment = UCRExperiment(name="UCR DB Experiment", context=experiment_context)
+    with experiment.control() as c:
```

Comment:
Running the sql results again. I figured this would make it less noisy in case the db changes in between when they look at the UCR and when the task is run